code/game/machinery/OpTable.dm

```diff
@@ -333,3 +333,71 @@
 		return TRUE
 	if(default_part_replacement(user, attacking_item))
 		return TRUE
+
+/obj/machinery/optable/robotics
```

Comment:
This should buckle people upright facing forwards rather than lying down like a normal optable if it doesn't already

Reply:
yep, it does that